examples/randomized-benchmark/two-qubit-rb/example_2qb_rb.py

```diff
@@ -0,0 +1,232 @@
+from rb_2qb import *
+from configuration import config
+from qm.QuantumMachinesManager import SimulationConfig
+from entropylab_qpudb import Resolver
+
+qmm = QuantumMachinesManager()
+
+"""
+For the required two qubit gates to generate the 4 classes (see Supplementary info of this paper:
+https://arxiv.org/pdf/1210.7011.pdf), we require the user to complete the following macros below according
+to their own native set of qubit gates, that is perform the appropriate decomposition and convert the pulse sequence
+in a sequence of baking play statements (amounts to similar structure as QUA, just add the prefix b. before every statement)
+Example :
+in QUA you would have for a CZ operation:
+    play("CZ", "coupler")
+in the macros below you write instead:
+    b.play("CZ", "coupler")
+"""
+# Define here quantum elements required to compute the macros
+q0 = "q0"
+q1 = "q1"
+coupler = "coupler"
+rr1 = "rr1"
+rr2 = "rr2"
+"""
+Define here your Resolver methods such that 
+qubit names "q0" and "q1" are mapped to the correct quantum elements in input config associated 
+to the two qubits you want to benchmark.
+The Resolver allows you to retrieve easily the names of the quantum elements associated to your target qubits, couplers,
+and other elements you might need to carry (fluxlines, etc...).
+
+In this example, we have the simplest case where "q0" and "q1" are defined as the two qubits directly in the config,
+and there is one single coupler named "coupler". Readout resonators coupled to each qubit are named "rr1" and "rr2"
+"""
+
+
+class RBTwoQubitResolver(Resolver):
+    def __init__(self, aliases: Union[dict, None] = None):
+        super().__init__(aliases)
+
+    def q(self, qubit, channel=None):
+        if qubit == "q0":
+            """
+            Put here all commands related to the targeted channel (eg. using two different elements for addressing the 
+            qubit with two distinct frequencies, this can be specified through the channel argument, or any additional 
+            that might be required)
+            """
+            return qubit
+
+        elif qubit == "q1":
+            return qubit
+        else:
+            return qubit
+
+    def res(self, resonator):
```

Comment:
same question as above

Reply:
Same answer as above, simplicity of our example config yields no specific case for resolver usage. Maybe we should pick a more complex architecture to showcase better the resolver usage. We should discuss which architecture should be addressed (Google perhaps?)